lib/web_ui/skwasm/canvas.cpp

```diff
@@ -7,8 +7,22 @@
 #include "helpers.h"
 #include "wrappers.h"
 
+#include "third_party/skia/include/core/SkPoint3.h"
+#include "third_party/skia/include/utils/SkShadowUtils.h"
+
 using namespace Skwasm;
 
+namespace {
+// These numbers have been chosen empirically to give a result closest to the
+// material spec.
```

Comment:
Do we need to keep these constants in sync with the mobile engine? If so, a link to the source of the mobile numbers could be useful.